README.md

```diff
@@ -94,93 +94,22 @@ What it sets up
 ---------------
 
 * [Bundler] for managing Ruby libraries
-* [Exuberant Ctags] for indexing files for vim tab completion
-* [Foreman] for managing web processes
```

Comment:
I think people still use foreman?

Reply:
fun fact this wasn't actually installing foreman! just readme cleanup, but the `pg` `bin/setup` (or some setup file) does install foreman there

Reply:
bahahah nice. good catch!